components/datetime/src/options/components.rs

```diff
@@ -94,8 +94,6 @@ pub struct Bag {
     /// Include the month, such as "April" or "Apr".
     pub month: Option<Month>,
     /// Include the week, such as "1st" or "1".
```

Comment:
Question: Is this enough documentation? I could see a user mistaking this for `weekday`, as at least in the `en` locale this would not be a common choice.

Reply:
I've changed the doc to use the term "week number" and updated the example to a number that cannot be conflated with a weekday ordinal.